tshirts.py

```diff
@@ -1,14 +1,24 @@
 
-def size(cms):
-    if cms < 38:
-        return 'S'
-    elif cms > 38 and cms < 42:
-        return 'M'
-    else:
-        return 'L'
+def get_size(cms):
+    size = None   
+    if cms >= 36 and cms < 38:
+        size = 'S'
+    elif cms >= 38 and cms < 42:
+        size = 'M'
+    elif cms >=42 and cms < 44:
+        size = 'L'
+    return size
 
 
-assert(size(37) == 'S')
-assert(size(40) == 'M')
-assert(size(43) == 'L')
+assert(get_size(37) == 'S')
+# added test to verify size of 38 cms
+assert(get_size(38) == 'M')

```

Comment:
you have added 36 and 44 as new boundary values. Ideally this is a requirement and before moving to implementation , it should have been captured as an expectation in test case. You could still update your tests!!